src/PowerShellEditorServices/Services/DebugAdapter/Debugging/VariableDetails.cs

```diff
@@ -168,12 +168,22 @@ private static string GetValueStringAndType(object value, bool isExpandable, out
             }
 
             Type objType = value.GetType();
-            typeName = $"[{objType.FullName}]";
+
+            // This is the type format PowerShell users expect and will appear when you hover a variable name
+            typeName = '[' + value.GetType().FullName + ']';
```

Comment:
```suggestion
            typeName = '[' + objType.FullName + ']';
```

We may actually even want to use `ToStringCodeMethods(Type)` here. Handles type accelerators and not assembly qualifying generic arguments. Probably out of scope though

Reply:
Thanks, I originally removed objType but then realized why it needed to be there but didn't go back and fix it here.